### PR TITLE
Add Multi From Demo

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Create Multi File demo based on Alex Ellis blog and Ben Breards blog.

### CODE REVIEW

File: demos/buildah_multi_stage.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+buildah images
+read -p "${green}Let's clear out our containers${reset}"
+buildah rm -a
+
+read -p "${green}Let's build the container with Buildah, first GoLang${reset}"
+buildcntr=$(buildah from golang:1.7.3)
+read -p "${green}Let's mount the container getting the root directory${reset}"
+buildmnt=$(buildah mount $buildcntr) 
+read -p "${green}Let's get x/net/html into the container${reset}"
+buildah run $buildcntr go get -d -v golang.org/x/net/html
```
Comment (TomSweeneyRedHat) [inline demos/buildah_multi_stage.sh]: For some reason after this line, the read commands are not stopping.  Everything runs, but no control by the user.  @ipbabble any thoughts?

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
LGTM
@ipbabble @pixdrift PTAL

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**